libs/designsystem/src/lib/components/tabs/tab-button/tab-button.component.scss

```diff
@@ -37,6 +39,15 @@ ion-tab-button {
     .icon-container {
       margin-right: utils.size('xxs');
     }
+
+    @include utils.slotted(div kirby-badge) {
+      &.md {
+        // Magic values here as we have no tokens that gives
+        // badge the correct position in relation to the tab icon
+        --kirby-badge-top: -1em;
+        --kirby-badge-left: 1.4em;
```

Comment:
You _could_ argue that this is redundant since you're declaring `--kirby-badge-left` with the same value as it already had - but I think it's also ok to leave it for clarity.